package.json

```diff
@@ -143,7 +144,10 @@
   "stylelint": {
     "extends": [
       "./config/stylelint-config-carbon"
-    ]
+    ],
+    "rules": {
+      "max-nesting-depth": 3
+    }
```

Comment:
As posted elsewhere, we should remove this I think

Reply:
Absolutely correct this isn't required- `**/packages/web-components/scss/**/*.scss` at [.stylelintignore](https://github.com/carbon-design-system/carbon/pull/17770/files#diff-36ae73c99ebd1b4072a3164e014df72eb094703df4548dd51938aa7faaa7f8d2) covers for web-component package